nf_core/utils.py

```diff
@@ -339,22 +339,22 @@ def setup_requests_cachedir():
     return config
 
 
-def wait_cli_function(poll_func, poll_every=20):
+def wait_cli_function(poll_func, refresh_per_second=20):
     """
     Display a command-line spinner while calling a function repeatedly.
 
     Keep waiting until that function returns True
 
     Arguments:
        poll_func (function): Function to call
-       poll_every (int): How many tenths of a second to wait between function calls. Default: 20.
+       refresh_per_second (int): Refresh this many times per second. Default: 20.
 
     Returns:
        None. Just sits in an infite loop until the function returns True.
     """
     try:
         spinner = Spinner("dots2", "Use ctrl+c to stop waiting and force exit.")
-        with Live(spinner, refresh_per_second=20):
+        with Live(spinner, refresh_per_second=refresh_per_second):
```

Comment:
I'm not sure if we even need to parameterise this. It's how often the _spinner_ refreshes and the actual polling runs every 2 seconds with the hardcoded `sleep` call. I think we can just hardcode the 20 fps spinner.

Reply:
But then why the argument in the function? If this is hardcoded the argument is mute - no?

Reply:
https://github.com/nf-core/tools/blob/f3160d096368ae2b3118562276b5248011f59907/nf_core/utils.py#L342